packages/wrangler/src/config/validation.ts

```diff
@@ -58,6 +58,14 @@ export function normalizeAndValidateConfig(
     } configuration:`
   );
 
+  deprecated(
+    diagnostics,
+    rawConfig,
+    "miniflare",
+    "Wrangler does not use configuration in the `miniflare` section. Unless you are using the Miniflare binary directly you can remove this section.",
```

Comment:
This isn't technically a "deprecation" since we never used it, . Also I think we shouldn't use the word "binary".

Reply:
Yeah - I was at 6s and 7s about calling it a deprecation - but the helper was just so nice to use...
Do you have a suggested alternative to the word "binary"? I wanted to make it clear that you have to be using miniflare directly yourself (outside of Wrangler) rather than knowing that `dev --local` happens to use miniflare internally).